javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,28 @@
+var width = 10;
+var height = 10;
+var area = height * width;
+console.log('Width x Height =', area);
+console.log('Typeof area=', typeof area);
+var payment = 90;
+var bill = 50;
+var change = payment - bill;
+console.log('The change is:', change);
+console.log('Typeof change:', typeof change);
+var quizzes = 10;
+var midterm = 20;
+var final = 30;
+var grade = (quizzes + midterm + final) / 3;
+console.log('The grade is:', grade);
+console.log('Typeof grade:', typeof grade);
+var firstName = 'Nelson';
+var lastName = 'Daza';
+var fullName = firstName + ' ' + lastName;
+console.log('The full name is:', fullName);
+console.log('Typeof fullName:', typeof fullName);
+var headCount = 30;
+var isSparta = (headCount === 300);
+console.log('The headcount is equal to Sparta:', isSparta);
+console.log('Typeof isSparta:', typeof isSparta);
+var motto = 'Nelson Daza' + ' ' + 'is the GOAT';
```

Comment:
You should use the `fullName` variable here, rather than just writing in the value yourself